### ISSUE TITLE
Support categories customization

### ISSUE THREAD

**Issue by tzarebczan** [OdyseeTeam/odysee-frontend#51]
The idea is that there would be some optional categories that users can enable in settings. Or they can turn off the default ones if they are not interested. This would also affect the homepage.

**Comment by tzarebczan** [comment]
@drewhancock can you add some details from our chat here on how we want this to work?

**Comment by drewhancock** [comment]
Yeah, so instead of having all the categories on the sidebar automatically, we'd have an Explore option on the sidebar that would take you to a page showing them all. Like what YouTube is doing here, just with more categories. 
![image](https://user-images.githubusercontent.com/80921344/168608531-548b0e43-ac2e-47de-9d26-1300aed43fca.png)

We'd default to showing a few basic ones on the homepage like music, education, gaming, and tech, and we'd allow users to add, remove, and reorder categories as they want.

Display wise, your Explore could look exactly like or pretty similar to how our current homepage does, while your real homepage is just a customized version of it, if that makes sense.
